contrib_bots/lib/xkcd.py

```diff
@@ -0,0 +1,139 @@
+from random import randint
+
+import logging
+import requests
+
+XKCD_TEMPLATE_URL = 'http://xkcd.com/%s/info.0.json'
+LATEST_XKCD_URL = 'http://xkcd.com/info.0.json'
+
+
+class XkcdHandler(object):
+    '''
+    This plugin provides several command that can be used for fetch comics from
```

Comment:
I'm also not sure it's `comics` - I'd use `comic strips` and `a comic strip` to be sure.